From 80c0a135afe5d801903ac742c386a5468837175c Mon Sep 17 00:00:00 2001
From: Dave Reisner <dreisner@archlinux.org>
Date: Wed, 2 Oct 2013 07:46:24 -0400
Subject: [PATCH] fix lingering references to /var/lib/{backlight,random-seed}

This should have been part of ef5bfcf668e6029faa78534dfe.
---
 man/systemd-backlight@.service.xml  | 2 +-
 man/systemd-random-seed.service.xml | 2 +-
 units/systemd-backlight@.service.in | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/man/systemd-backlight@.service.xml b/man/systemd-backlight@.service.xml
index 2b73625..4318964 100644
--- a/man/systemd-backlight@.service.xml
+++ b/man/systemd-backlight@.service.xml
@@ -58,7 +58,7 @@
                 is a service that restores the display backlight
                 brightness at early-boot and saves it at shutdown. On
                 disk, the backlight brightness is stored in
-                <filename>/var/lib/backlight/</filename>. Note that by
+                <filename>/var/lib/systemd/backlight/</filename>. Note that by
                 default, only firmware backlight devices are
                 saved/restored.</para>
         </refsect1>
diff --git a/man/systemd-random-seed.service.xml b/man/systemd-random-seed.service.xml
index 8cd14b7..e5cd037 100644
--- a/man/systemd-random-seed.service.xml
+++ b/man/systemd-random-seed.service.xml
@@ -61,7 +61,7 @@
                 for details. Saving/restoring the random seed across
                 boots increases the amount of available entropy early
                 at boot. On disk the random seed is stored in
-                <filename>/var/lib/random-seed</filename>.</para>
+                <filename>/var/lib/systemd/random-seed</filename>.</para>
         </refsect1>
 
         <refsect1>
diff --git a/units/systemd-backlight@.service.in b/units/systemd-backlight@.service.in
index b0e75db..5caa5d5 100644
--- a/units/systemd-backlight@.service.in
+++ b/units/systemd-backlight@.service.in
@@ -9,7 +9,7 @@
 Description=Load/Save Screen Backlight Brightness of %I
 Documentation=man:systemd-backlight@.service(8)
 DefaultDependencies=no
-RequiresMountsFor=/var/lib/backlight
+RequiresMountsFor=/var/lib/systemd/backlight
 Conflicts=shutdown.target
 After=systemd-readahead-collect.service systemd-readahead-replay.service systemd-remount-fs.service
 Before=sysinit.target shutdown.target
-- 
1.8.4.652.g0d6e0ce

